installer_test.go

```diff
@@ -0,0 +1,32 @@
+package gowin32
+
+import (
+	"testing"
+)
+
+func TestGetInstalledProducts(t *testing.T) {
```

Comment:
Our first unit test!  A lot of the stuff in this library is hard to test without modifying system settings, but this works fine, so I'm certainly not complaining.